src/Components/Onboarding/Steps/CollectorIntent.tsx

```diff
@@ -24,12 +25,12 @@ interface State {
 
 class CollectorIntent extends React.Component<Props, State> {
   options = [
-    "Buy Art & Design",
-    "Sell Art & Design",
-    "Research Art Prices",
-    "Learn About Art",
-    "Find Out About New Exhibitions",
-    "Read Art Market News",
+    "buy art & design",
```

Comment:
Should these be updated to match the enums in Metaphysics?

ie- `"BUY_ART_AND_DESIGN"`